doc/api/fs.markdown

```diff
@@ -1157,6 +1157,17 @@ reliably or at all.
 You can still use `fs.watchFile`, which uses stat polling, but it is slower and
 less reliable.
 
+#### Inodes
+
+<!--type=misc-->
+
+On Linux and OS X systems, `fs.watch` resolves the watched path to an inode. If
+the watched path is deleted and then created so that it has a new inode, you
```

Comment:
We encourage to avoid it. Redacted myself often about this too.
I'd see this as a nit, but worthwhile.
https://github.com/nodejs/docs/blob/master/STYLE-GUIDE.md